android/src/main/java/org/ligi/passandroid/ui/quirk_fix/URLRewriteActivity.java

```diff
@@ -16,7 +16,7 @@ protected void onCreate(Bundle savedInstanceState) {
         final Uri data = getIntent().getData();
         final String url = data != null ? new URLRewriteController(getTracker()).getUrlByUri(data) : null;
 
-        if (data != null && url == null) {
+        if ((data != null && url == null) || data == null) {
```

Comment:
I think this can be simplified to  url == null || data == null

Reply:
If data is null, then url is also null (see line above). Can even be simplified further:
```suggestion
        if (url == null) {
```